packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -355,7 +394,8 @@ export default class ElectronAppWrapper {
 		const alreadyRunning = this.ensureSingleInstance();
 		if (alreadyRunning) return;
 
-		this.createWindow();
+		// Loads the Splash Screen and createWindow is called from the below function
+		this.splashScreen();
```

Comment:
Make splahScreen() return the created window. And at this point add call createWindow() and add the event listener. It will make the dependencies clearer

Reply:
Resolved as asked